docs/indexing_samples.md

```diff
@@ -0,0 +1,51 @@
+# Adding sample indexing to the index runner
+
+## Reusable code
+
+* The event loop in `main.py`
+* The logging code in `main.py`
+* `config.py` (currently some code in `main.py` could probably be moved into `config.py` for
+  separation of concerns reasons)
+* The Kafka code in `main.py` with configurable topics
+
+## Non-reusable code
+
+* `_handle_message` in `main.py` and the methods it calls
```

Comment:
Do you mean `_handle_msg`? How/why are you wanting to reuse that?

Reply:
Yes. I *don't* want to reuse that, I don't think it's useful for samples